script.js

```diff
@@ -0,0 +1,51 @@
+const bookName = document.getElementById('name');
+const author = document.getElementById('author');
+const addBtn = document.querySelector('.addBtn');
+const output = document.querySelector('.list-elements');
+let arr = [];
```

Comment:
- [ ] Your app should contain a collection that keeps the list of books. I would suggest populating the array with objects that contain data about books as a default value instead of being empty.

Reply:
Hey Omar,
Thanks for giving me an review.
I am unable to understand what you want to say. Can you elaborate your review changes
Thanks for your review again.
Best Regards,
9:18
Hammas Bin Saleem

Reply:
Thanks for reaching out to me. I have implemented your suggestion in my JavaScript.